code/modules/mob/living/simple_animal/vore/rous.dm

```diff
@@ -43,4 +43,117 @@
 
 /mob/living/simple_animal/hostile/rous/death()
 	playsound(src, 'sound/effects/mouse_squeak_loud.ogg', 50, 1)
+	if(resting)
+		resting = 0
+		icon_state = icon_dead
+		update_icon()
 	..()
+
+/mob/living/simple_animal/hostile/rous/passive
```

Comment:
Also why is it called rous? The hell is a rous?

Reply:
oh my god have you not seen the princess bride

Rodent Of Unusual Size

Reply:
Also shorter than "giant_rat" and less misleading than simple "rat" heh

Reply:
I'm going to change this because admins aren't going to know that innately when trying to spawn them.